src/components/welcome/Welcome.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+import './Welcome.css';
+import { useNavigate } from 'react-router-dom';
+
+const Welcome: React.FC = () => {
+  const [loading, setLoading] = useState(true);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const token = localStorage.getItem('token');
+    if (!token) {
+      navigate('/login');
+      return;
+    } else {
+      const timeout = setTimeout(() => {
+        setLoading(false);
+      }, 2000);
+
+      return () => clearTimeout(timeout);
+    }
+  }, [navigate]);
+
+  return <>{loading ? <div className='custom-loader-page'></div> : <div>Welcome</div>}</>;
```

Comment:
```suggestion
  return <>{loading ? <div className='custom-loader-page' /> : <h3>Welcome</h3>}</>;
```

obs: h3 ou outro elemento que seja destinado a texto que você entende ser melhor.

Reply:
Substitui para a tag h3, para diferenciar com a tag div